_pages/en/capabilities.md

```diff
@@ -1,50 +1,221 @@
+
+##### 1.1.3 Goals
+
+- 100% of application code stored in version control system
+- 100% of system configurations stored in version control system
+- 100% of application configurations stored in version control system
+- 100% of scripts for automating build and configuration stored in version control system
+
+##### 1.1.4. How to improve
+
+- Save all the things in version control - not just application code and dependencies
+- Ensure that every commit to version control triggers the automated creation of packages that can be deployed to any environment using only information in version control.
+- Make it possible to create production-like test environments on demand using only scripts and configuration information from version control, and to create packages using the automated process described in the previous approach.
+- Work in the open on Github, Gitlab, NPM, Docker Hub, ..
+- Work on shared GC version control system (GCcode)
+- Work on official internal ESDC version control system (TFS, Azure Dev,,)
+
+#### 1.2 Continuous integration
+
+##### 1.2.1 Outcomes
+
+Teams must integrate all their work into the main version of the code base on a regular basis.
+Changes must trigger automated builds of the software and a series of automated tests that provide feedback in a few minutes.
+Builds should be numbered and repeatable.
+
+Tests should run successfully at least once a day.
+The system should also make the status visible to the team.
+
+##### 1.2.2. Indicators
+
+- What percentage of code commits result in a software build without manual intervention?
+- What percentage of code commits result in a suite of automated tests being run without manual intervention?
+- What percentage of automated builds and tests that are executed successfully every day?
+- Are builds available to testers?
+- What percentage of tests provide feedback that is available to developers within a day?
+- How long does it takes between the build breaking and having it fixed, either with a check-in that fixes the problem, or by reverting the breaking change?
+- Are builds numbered and repeatable?
+- Are automated tests run every day?
+- How long do tests and build take?
+
+How to measure?
+
+##### 1.2.3 Goals
+
+- All code commits result in a software build without manual intervention
+- All code commits result in a suite of automated tests being run without manual intervention
+- 85% of automated builds and tests that are executed successfully every day
+- All tests provide feedback that is available to developers within a day
+- Fixing build or failing tests takes [1-24] hours
+
+##### 1.2.4. How to improve
+
+- Use version control
+- Remove manual build steps - automate
+- Trigger quick tests on every change (unit tests) for quick feedback
+- Make fixing a broken build a priority
+- Limit tests to 10 minutes
+- Merge changes frequently into trunk
+
+### 2. Process Capabilities
+
+#### 2.2 Streamlining change approval
+
+##### 2.2.1 Outcomes
+
+...
```

Comment:
The organization has built a clear and extensive set of guardrails to accelerate and scale the design and delivery of IT solutions.
Teams are empowered to take decisions for the vast majority of the cases, only ever requiring approvals for exceptions.

Reply:
Automation for compliance has been introduced for all teams so that focus is put on empowering teams, not gating the development of solutions.